ktx-library-codelab/step-03/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<resources>
+    <string name="app_name">KotlinKtxWorkshop</string>
```

Comment:
Maybe KtxLibraryCodelab to keep the same name as the project
Prefer Codelab over Workshop in any case